### PR TITLE
Add KL weight annealing to scvi

### LINKED ISSUE #316: scvi: enable KL annealing
Enable a cellarium model (in a way that plays nicely with lightning) to anneal some prefactor in a loss function -- in this case, a prefactor for the KL divergence in the loss.

It should be able to follow some kind of a schedule during training (like a linear ramp-up to 1 in our case, over a defined number of epochs).

Figure out the RIGHT way to do this in lightning.  Look for examples.

### PR DESCRIPTION
**Pull request by sjfleming**
Closes #316 

Add KL annealing to the scvi model.

This hard-codes the addition of "epoch" to `batch`, which we probably don't want to do for all models all the time.  But for scvi, we'd like to run without specifying `"test_mode"`.  But dadc "test_mode" already attaches "epoch" to the batch.

### CODE REVIEW

**Comment by sjfleming** [comment]
Do not merge yet... there seems to be a regression introduced here
